### PR TITLE
Data Publication release for 2022

### PR DESCRIPTION
**Pull request by Michaeldremy**
### Releases various datasets and makes updates to various data browser applications.

Datasets being released:

- 2022 Snapshot
- 2022 Aggregate & Disclosure reports
- 2021 One Year
- 2019 Three Year

Data browser update for 2022:

- Dataset filtering tool
- Maps

Data publication Cypress tests were updated to include entries for 2022:

- Dataset filtering tool
- Maps
- Aggregate & Disclosure reports

### CODE REVIEW

**Review by ojbravo — APPROVED** [review]
Lots of changes, but looks clean. Well done.

File: src/data-publication/constants/one-year-datasets.js [hunk 1]
```diff
@@ -1,4 +1,33 @@
 export const ONE_YEAR_DATASET = {
+  2021: {
+    freezeDate: 'May 1, 2022',
```
Comment (meissadia) [inline src/data-publication/constants/one-year-datasets.js:3]: I think this was also corrected somewhere else
```suggestion
    freezeDate: 'May 1, 2023',
```

**Review by meissadia — APPROVED** [review]
One potential date correction but everything else looks good.  Awesome work @Michaeldremy!

File: src/data-publication/constants/one-year-datasets.js (hunk 1 above)
Comment (meissadia) [reply src/data-publication/constants/one-year-datasets.js:3]: @ojbravo Can you confirm this?

Comment (ojbravo) [reply src/data-publication/constants/one-year-datasets.js:3]: @meissadia Good catch! This was the same update Abhi cited as well.

**Pull request merged**